deployments/helm/onos-gui/templates/deployment.yaml

```diff
@@ -0,0 +1,117 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "onos-gui.fullname" . }}
```

Comment:
add namespace under the metadata. 
`namespace: {{ .Release.Namespace }}`

Reply:
done